system-addon/content-src/components/TopSites/_TopSites.scss

```diff
@@ -326,34 +320,89 @@
 }
 
 .topsite-form {
+  $form-width: 300px;
+  $form-spacing: 32px;
+
+  .form-input-container {
+    max-width: $form-width + 3 * $form-spacing + $rich-icon-size;
+    margin: 0 auto;
+    padding: $form-spacing $form-spacing 40px;
+
+    .top-site-outer {
+      padding: 0;
+      margin: 24px 0 0;
+      margin-inline-start: $form-spacing;
+      pointer-events: none;
+      cursor: default;
+    }
+
+    .section-title {
+      text-transform: none;
+      font-size: 16px;
+      line-height: 19px;
+      color: $grey-60;
+      margin: 0 0 16px;
+    }
+  }
+
+  .fields-and-preview {
+    display: flex;
+  }
+
+  label {
+    font-size: $section-title-font-size;
+  }
+
   .form-wrapper {
-    margin: auto;
-    max-width: 350px;
-    padding: 15px 0;
+    width: 100%;
 
     .field {
       position: relative;
+
+      .clear-input-value-container {
+        position: absolute;
+        top: 16px;
+        width: 24px;
+        background: $white;
+        pointer-events: none;
+
+        &:dir(rtl) {
+          offset-inline-start: 1px;
```

Comment:
This probably isn't what you wanted:
![screen shot 2018-02-16 at 3 00 46 pm](https://user-images.githubusercontent.com/438537/36333192-4c65e122-132a-11e8-9bd6-c44f6e4d8a7c.png)

Just have an `offset-inline-end: 1px` without a `:dir` check. It'll be as if it's `left: 1px` for `rtl` (and `right: 1px` for `ltr`).

Reply:
You can test by opening up the inspector and changing the top level `<html dir="rtl">`

Reply:
I did know about the `.rtl` class but I never actually typed something while in rtl mode and it looked ok to me..